power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/userRegistrationController.java

```diff
@@ -0,0 +1,52 @@
+package com.adorsys.gis.powerpay.powerpaybackend.services;
+
+import com.adorsys.gis.powerpay.powerpaybackend.domain.User;
+import com.adorsys.gis.powerpay.powerpaybackend.domain.UserRegistration;
+import jakarta.validation.Valid;
+import lombok.Getter;
+import lombok.Setter;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/registration")
+public class userRegistrationController {
```

Comment:
You better use [@ResponseBody](https://www.baeldung.com/spring-request-response-body) for simplicity and use [@ResponseStatus](https://www.baeldung.com/spring-response-status) to add status behavior.

Reply:
Done